STEP_Project/tests/OUSelectContentTest.js

```diff
@@ -0,0 +1,77 @@
+let tabl;
+let orgUnits;
+let numRows;
+
+describe('Testing selectClick function called when "select" button is clicked', ()=>
+{
+    let mockChrome;
+    beforeAll(function()
+    {
+        //initialize mock storage to mimic chrome.storage.sync
+        mockChrome = 
+        {
+            dict: {},
+            get : function(arg1, arg2)
+            {
+                arg2(this.dict);
+            },
+            set : function(pair)
+            {
+                for (let key in pair)
+                {
+                    mockChrome.dict[key] = pair[key];
+                }
+            },
+            remove : function(dataid)
+            {
+                this.dict[dataid]= undefined;
+            }
+        };
+
+        //initialize the UL to mimic the DOM of the settings page
+        numRows = 3;
+        tabl = document.createElement('ul');
+        tabl.setAttribute("role", "group");
+        for (let i = 0; i < numRows; i++)
+        {
+            const liObj = document.createElement('li');
+            const entry = document.createElement('div');
+            liObj.innerText = "row"+i;
+            entry.setAttribute("data-node-id", i);
+            liObj.appendChild(entry);
+            tabl.appendChild(liObj);
+            mockChrome.set({[i]:liObj.innerText})
+        }
+        document.body.appendChild(tabl);
+        storageObj = mockChrome;
+    })
+
+    afterAll(function()
+    {
+        tabl.remove();
+    })
+    
+    it('Should update the mock storage if entity\'s name is changed', ()=>
```

Comment:
If I understand correctly, this actually tests what was done in https://github.com/googleinterns/preferred-entity-selector/pull/17, right ? If yes, mention that too in your PR.

Reply:
Yes, I updated the description.